.github/workflows/docker_latest.yml

```
@@ -1,212 +1,212 @@
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+  update_latest_branch:
+    name: Update latest branch
+    runs-on: ubuntu-22.04
+    needs: push_to_registry
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ inputs.latest_branch }}
```

actionlint (1.7.12)

[error] property "latest_branch" is not defined in object type {branch: string; disable_tests: bool; platforms: string; tag: string}

(expression)